cmd/network/setup_linux.go

```diff
@@ -99,6 +92,82 @@ func main() {
 		logrus.Fatal(err)
 	}
 
+	connFile, err := vsockConn.File()
+	if err != nil {
+		logrus.Fatal(err)
+	}
+
+	vmSwitchCmd := configureVMSwitch(ns,
+		vmSwitchLogFile,
+		vmSwitchPath,
+		tapIface,
+		subnet,
+		tapDeviceMacAddr,
+		connFile)
+
+	if err := vmSwitchCmd.Start(); err != nil {
```

Comment:
We don't actually need to declare a new `err` here (we got one on line 95). But this is fine too.